sklearn/metrics/_plot/tests/test_roc_curve_display.py

```diff
@@ -125,6 +125,79 @@ def test_roc_curve_display_plotting(
     assert display.ax_.get_xlabel() == expected_xlabel
 
 
+@pytest.mark.parametrize("plot_chance_level", [True, False])
+@pytest.mark.parametrize(
+    "chance_level_kwargs",
+    [
+        {"linewidth": 1},
+        {"color": "b", "label": "DummyEstimator"},
+        {"color": "g", "linestyle": ":"},
```

Comment:
I would limit only to 2 possibilities: `None` for the default and one dictionary. It would be enough.